justfile

```diff
@@ -53,5 +53,21 @@ test-cluster:
   # No test instance for this crd
   cargo build --test runner
 
+test-linkerd-serverauth:
+  kubectl apply --server-side -f tests/serverauth-crd.yaml
+  cargo run --bin kopium -- --docs serverauthorizations.policy.linkerd.io > tests/gen.rs
+  echo "pub type CR = ServerAuthorization;" >> tests/gen.rs
+  kubectl apply -f tests/serverauth.yaml
+  cargo test --test runner -- --nocapture
+
+# This fails in the test runner when trying to patch the applied instance.
+# let _obj = cr.patch_status("gen", &pp, &patch).await?;
+#test-linkerd-server:
+#  kubectl apply --server-side -f tests/server-crd.yaml
+#  cargo run --bin kopium -- --docs servers.policy.linkerd.io > tests/gen.rs
+#  echo "pub type CR = Server;" >> tests/gen.rs
+#  kubectl apply -f tests/server.yaml
```

Comment:
you have run into an edge case of the test runner. if you name it something other than `server` the `path_status` branch won't happen.

..ideally, i should refactor that branch to only trigger if the file with the `gen` object has a status object, but never got that far.

Reply:
I just pushed a change to skip that if the file doesn't contain the status object.

Reply:
ah, that's great. thanks a lot!